src/backend/PhotoAPI/API/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+
+namespace API
+{
+    public class Program
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=4794247)